src/overlays/actors/ovl_En_Water_Effect/z_en_water_effect.h

```diff
@@ -19,11 +19,11 @@ typedef struct {
 } EnWaterEffectStruct; // size = 0x40
 
 enum {
-    /* 0x  1 */ ENWATEREFFECT_1 = 1,
-    /* 0x309 */ ENWATEREFFECT_777 = 777,
-    /* 0x30A */ ENWATEREFFECT_778,
-    /* 0x30B */ ENWATEREFFECT_779,
-    /* 0x30C */ ENWATEREFFECT_780,
+    /* 001 */ ENWATEREFFECT_1 = 1,
+    /* 777 */ ENWATEREFFECT_777 = 777,
+    /* 778 */ ENWATEREFFECT_778,
+    /* 779 */ ENWATEREFFECT_779,
+    /* 780 */ ENWATEREFFECT_780,
 };
```

Comment:
The point of using hex on enums is to make it easy to see in memory or in the code, don't think this is an appropriate change; if anything the numbers should change to reflect the hex values.